src/acceptance-testing/release-readiness.md

```diff
@@ -0,0 +1,125 @@
+# Release Readiness Process (RRP)
+
+## Overview
+
+This document defines the process and expectations for devnet releases in the OP Stack. It establishes a consistent framework for determining when a devnet is ready for release, how results are communicated, and how pass/fail determinations are made. By following these procedures, we can ensure that devnets meet quality standards before promotion, reducing the risk of critical bugs reaching production networks.
+
+## Roles and Responsibilities
+
+While the Platforms team serves as the primary custodian of this release readiness process, its success relies on collaborative ownership with Protocol and contributions from across the organization.
+
+| Role | Responsibilities |
+|------|-----------------|
+| **Platforms Team** | • Maintain the Release Readiness Process (RRP)<br>• Run acceptance tests<br>• Make final pass/fail determinations<br>• Communicate test results to stakeholders |
+| **Feature Teams** | • Write acceptance tests for specific features<br>• Run feature-specific tests<br>• Fix identified issues in their features<br>• Provide input on severity of identified issues |
```

Comment:
I feel like a feature shouldn't be included in a devnet if it doesn't already have acceptance tests written and shown to be passing.

Reply:
Agreed!
Platforms may not always be able to notice the lack of test coverage for new features however, so this will need to be a team effort. We should set this expectation and all remain vigilant to any erroneous diversions.